apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx

```diff
@@ -195,6 +194,41 @@ export const InboundPassiveStsForm: FunctionComponent<InboundPassiveStsFormProps
                         </Hint>
                     </Grid.Column>
                 </Grid.Row>
+                <Grid.Row columns={ 1 }>
+                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                        <Field
+                            name="replyToLogout"
+                            label={ 
+                                t("console:develop.features.applications.forms.inboundSTS.fields.replyToLogout.label") 
+                            }
+                            required={ true }
```

Comment:
We don't need to explicitly set boolean value for props whose value always remain `true`.

```suggestion
                            required
```

Reply:
[0f5a444](https://github.com/wso2/identity-apps/pull/4518/commits/0f5a444d9f777190086315c82c4099bd5b6dd961)